src/tests/socket.test.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable mocha/no-top-level-hooks */
```

Comment:
it's better to remove this eslint-disable

Reply:
Yes sure, resolved.